public/index.html

```diff
@@ -39,5 +39,20 @@
       To begin the development, run `npm start` or `yarn start`.
       To create a production bundle, use `npm run build` or `yarn build`.
     -->
+    <script
+      src="https://code.jquery.com/jquery-3.3.1.slim.min.js"
+      integrity="sha384-q8i/X+965DzO0rT7abK41JStQIAqVgRVzpbzo5smXKp4YfRvH+8abtTE1Pi6jizo"
+      crossorigin="anonymous"
+    ></script>
+    <script
+      src="https://cdnjs.cloudflare.com/ajax/libs/popper.js/1.14.7/umd/popper.min.js"
+      integrity="sha384-UO2eT0CpHqdSJQ6hJty5KVphtPhzWj9WO1clHTMGa3JDZwrnQq4sF86dIHNDz0W1"
+      crossorigin="anonymous"
+    ></script>
+    <script
+      src="https://stackpath.bootstrapcdn.com/bootstrap/4.3.1/js/bootstrap.min.js"
+      integrity="sha384-JjSmVgyd0p3pXB1rRibZUAYoIIy6OrQ6VrjIEaFf/nJGzIxFDsf4x0xIM+B07jRM"
+      crossorigin="anonymous"
+    ></script>
```

Comment:
I don't like the idea mixing the old-school way of important scripts because its harder to manage version updates and taking extra time to block rendering. 

If we need to pull in jquery, bootstrap.js, and popper.js for one simple UX feature, is there a more react way to do this? Could we there be a way to pull these dependencies in only on the component via imports? Or maybe using a reactified bootstrap component from another library?

Reply:
Some links on this:
- [Adding Bootstrap to React from the React docs](https://facebook.github.io/create-react-app/docs/adding-bootstrap)
- [React-bootstrap library for popovers](https://react-bootstrap.github.io/components/overlays/#popovers)
- [reactstrap popovers](https://reactstrap.github.io/components/popovers/)
- [react wrapper for popover.js](https://github.com/FezVrasta/react-popper)

Reply:
Thanks for pointing me in this direction. I'm with you after reading through that first post. I ended up replacing this feature with components from react-strap and was able to salvage the existing styles and drop them in. Everything looks the same, and it was good to try out reactstrap since I've only used React Bootstrap before.